### PR TITLE
[SMALLFIX]javadoc for ContainerAllocator#requestContainers

### PR DESCRIPTION
**Pull request by ParanoiaW**

### CODE REVIEW

**Comment by gpang** [comment]
Jenkins, test this please

File: integration/yarn/src/main/java/alluxio/yarn/ContainerAllocator.java [hunk 1]
```diff
@@ -139,6 +139,9 @@ public synchronized void allocateContainer(Container container) {
     mOutstandingContainerRequestsLatch.countDown();
   }
 
+  /**
+   * Requesets for containers.
```
Comment (gpang) [inline integration/yarn/src/main/java/alluxio/yarn/ContainerAllocator.java]: typo: Requesets -> Requests

Comment (ParanoiaW) [inline integration/yarn/src/main/java/alluxio/yarn/ContainerAllocator.java]: thanks for your comments and i have corrected this

File: integration/yarn/src/main/java/alluxio/yarn/ContainerAllocator.java [hunk 2]
```diff
@@ -139,6 +139,9 @@ public synchronized void allocateContainer(Container container) {
     mOutstandingContainerRequestsLatch.countDown();
   }
 
+  /**
+   * Requests for containers.
```
Comment (aaudiber) [inline integration/yarn/src/main/java/alluxio/yarn/ContainerAllocator.java]: The method name already says this. This is a private method so javadoc is not mandatory

**Pull request closed without merging**